.github/workflows/docker-push.yml

```diff
@@ -3,7 +3,10 @@ name: Create and publish a Docker image
 
 on:
   push:
-    branches: ['release']
+    branches: ['main']
```

Comment:
Is it possible to do this nigthly and not on every push to main?
Main gets a lot of pushes.